### PR TITLE
#182775937 Ft Create Tooltip Component 

### PR DESCRIPTION
**Pull request by RACHID-WEBDEV**
#### What does this PR do?
Create Tooltip Component 
#### Description of Task to be completed?
- Created the  `ToolTip Component`  
- Modified `GlobalStyle.css`, `Input Copmponent`  , `Select Component` and `Textarea Component`

#### How should this be manually tested?
- Clone the Repository
- Run `yarn Install` in your terminal to install required dependencies
-  Run `yarn dev` to start the local development server on `localhost:3000/form-component

### CODE REVIEW

File: components/form/FormIcons.js [hunk 1]
```diff
@@ -1,25 +1,47 @@
...
+  floatLabel,
+  toolTip,
+  isValid,
+  hasErrors,
+  toolTipTitle,
+  toolTipContent,
+  toolTipColor,
+  toolTipPlacement,
+  showTooltipArrow,
+  isTextArea,
```
Comment (andela-hpopoola) [inline components/form/FormIcons.js:17]: As I have always said, defining props for each element is not sustainable and would be difficult to manage in the future. 

For instance, if we are adding Tooltip to the Datepicker component, we will need to add the props here, so this is not feasible.

Let's remove this file and use the tooltip component directly. We can use the positioning of each element to determine how it should look like. 

I will explain better during our sync.

**Review by andela-hpopoola — CHANGES_REQUESTED** [review]
Thank you of working on this. I left a few comments but will explain more during our sync.

File: components/form/FormIcons.js (hunk 1 above)
Comment (RACHID-WEBDEV) [reply components/form/FormIcons.js:17]: Resolve now

File: components/form/Input.js [hunk 2]
```diff
@@ -2,11 +2,10 @@ import React, { useState } from 'react';
 import { useFormContext } from 'react-hook-form';
 import PropTypes from 'prop-types';
 import { ShowPasswordIcon, HidePasswordIcon } from '@/public/assets/signUp/passwordSvgs';
-import FormIcons from '@/components/form/FormIcons';
 import { Label, ErrorMessage } from '.';
 import classNames from 'classnames';
 import { showError, showSuccess } from '@/utils/form-helpers';
-import { ToolTipIcon } from '@/public/assets/dashboard/navBarIcon';
+import { HelpIcon, SuccessIcon } from '@/public/assets/dashboard/navBarIcon';
```
Comment (andela-hpopoola) [inline components/form/Input.js:8]: HelpIcon and SuccesIcon shouldn't be important from navBarIcon. There are no correlation between them, so we might create another file for them.

File: pages/form-component.js [hunk 3]
```diff
@@ -74,6 +74,7 @@ const SignIn = () => {
                             toolTipPlacement="topEnd"
                             // showTooltipArrow={false}
                             toolTipColor="light"
+                            //toolTipIcon={<EmailIcon />}
```
Comment (andela-hpopoola) [inline pages/form-component.js:77]: Lets remove all of the commented out codes 🙏🏽

**Review by andela-hpopoola — APPROVED** [review]
Thank you for helping out with this. It looks much better and should be ready for merging after you fixed the little feedback I left.

Nice work and thank you.

**Pull request merged**